### ISSUE TITLE
Module parse failed: Unexpected token "_?"

### ISSUE THREAD

**Issue by BennyLu-1** [mswjs/interceptors#327]
I'm getting this error when trying to use `ClientRequestInterceptor` or `XMLHttpRequestInterceptor`
Fetch works though.
It produces this error when running `const interceptor = new ClientRequestInterceptor();`
```
Error: Module parse failed: Unexpected token (120:13)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| 			}
| 
> 			response_?.emit("error", err);
| 		});
| 
```

Suggestions?

**Comment by kettanaito** [comment]
Hey, @BennyLu-1. Thanks for reporting this. 

It seems we've set the build `target` for this package to be "esnext", which is a bit optimistic. I will configure it back to `es6` so it could run in your project.

**Comment by kettanaito** [comment]
## Released: v0.19.5 🎉

This has been released in v0.19.5!

- 📄 [**Release notes**](https://github.com/mswjs/interceptors/releases/tag/v0.19.5)
- 📦 [npm package](https://www.npmjs.com/package/@mswjs/interceptors/v/0.19.5)

Make sure to always update to the latest version (`npm i @mswjs/interceptors@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
